ios/RNSpokestack.m

```diff
@@ -2,14 +2,14 @@
 #import "RNSpokestack.h"
 #import <React/RCTConvert.h>
 #import <React/RCTLog.h>
-#import <SpokeStack/SpokeStack.h>
+#import <SpokeStack/SpokeStack-Swift.h>
 
 @implementation RNSpokestack
 {
     bool hasListeners;
 }
 
-RCT_EXPORT_MODULE();
+RCT_EXPORT_MODULE(Spokestack);
```

Comment:
Wait if we are keeping the JS module named `RNSpokestack` doesn't this override that? @noelweichbrodt

Reply:
This is the name for the native module, no the js module. So the js import will be `const { Spokestack } = NativeModules`